packages/components/app/styles/components/alert.scss

```diff
@@ -63,9 +63,9 @@
     border-radius: 5px;
   }
 
-  // Notice: in the future this may become a "Link::Inline" component (for now we declare directly the same styles here)
-  a {
-    color: var(--token-color-foreground-action);
+  // Default styling for bare HTML links not using HDS::Link components
+  a:not([class*="hds-"]) {
```

Comment:
Thanks I agree but the comment comes across as a critique on the change I made. Thanks for clarifying though.

Reply:
Ah, apologies for that! I tend to use code review as an opportunity to reflect on existing code as well as the introduced changes. Will try to better separate the two in the future.

Reply:
@alex-ju (and @KristinLBradley) let me add some context here, that may reduce your apprehension regarding the styling of basic HTML tags (totally justified). 
When we implemented the `Alert/Toast` component, we realized that we needed to provide support for this kind of basic HTML elements (see https://docs.google.com/document/d/1MC0MTm8-0F26s6Y-gou_RsMj6rtKS6jC-eM7rmruqLY/edit#bookmark=id.jmn5w6np3e0z) so we decided it was relatively safe to scope/limit these styles in the `description` block of the `Alert/Toast` component (considered that we expose also a `Generic` container, it's reasonable to think that if a consumer needs to put more structured content (eg. a component using one of these tags) in the `Alert/Toast`, it's likely that will be done using the `Generic` container, not the `Description`).

This is also the reason why we added the CSS rule targeting the generic `a` element, because at that time we didn't know a `Link::Inline` component would be introduced later, with two different color variants.

With all of this in mind, I think is relatively safe to make the change that @KristinLBradley has made. The use of `:not()` makes me a bit nervous, but I can't see an alternative to it, so I'm going to hold my nose for this specific case 🙂